server/tests/mochData/accommodation.js

```diff
@@ -0,0 +1,66 @@
+export const validAccommodation = {
+  name: 'MILLE COLLINES HOTEL',
+  description: 'we can have you any time',
+  locationId: 1,
+  amenities: 'hey hey You will enjoy',
+  images: ['https://mcdn.wallpapersafari.com/medium/91/83/jECzOG.jpg', 'https://mcdn.wallpapersafari.com/medium/40/15/fmFJbU.jpg'],
+  rooms: [{
+    roomType: 'Single',
+    numberOfRooms: 5,
+    roomAmenities: 'hey hey You will enjoy',
+    roomImages: ['https://mcdn.wallpapersafari.com/medium/91/83/jECzOG.jpg', 'https://mcdn.wallpapersafari.com/medium/40/15/fmFJbU.jpg'],
+    cost: 600
+  },
+  {
+    roomType: 'Double',
+    numberOfRooms: 5,
+    roomAmenities: 'hey hey You will enjoy',
+    roomImages: ['https://mcdn.wallpapersafari.com/medium/91/83/jECzOG.jpg', 'https://mcdn.wallpapersafari.com/medium/40/15/fmFJbU.jpg'],
+    cost: 600
+  }]
+};
+
+export const invalidAccommodation = {
+  name: 5,
+  description: 'we can have you any time',
+  locationId: 1,
+  amenities: 'hey hey You will enjoy',
+  images: ['https://mcdn.wallpapersafari.com/medium/91/83/jECzOG.jpg', 'https://mcdn.wallpapersafari.com/medium/40/15/fmFJbU.jpg'],
+  rooms: [{
+    roomType: 'Single',
+    numberOfRooms: 5,
+    roomAmenities: 'hey hey You will enjoy',
+    roomImages: ['https://mcdn.wallpapersafari.com/medium/91/83/jECzOG.jpg', 'https://mcdn.wallpapersafari.com/medium/40/15/fmFJbU.jpg'],
+    cost: 600
+  },
+  {
+    roomType: 'Double',
+    numberOfRooms: 5,
+    roomAmenities: 'hey hey You will enjoy',
+    roomImages: ['https://mcdn.wallpapersafari.com/medium/91/83/jECzOG.jpg', 'https://mcdn.wallpapersafari.com/medium/40/15/fmFJbU.jpg'],
+    cost: 600
+  }]
+};
+
+
+export const existedAccommodation = {
```

Comment:
Rename the variable to **existingAccommodation** instead.

Reply:
Feedback implemented.